.changeset/loud-owls-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
This is gonna be an internal change that doesn't require any external changes, so should be a `patch` while we're on v0